packages/arb-token-bridge-ui/src/hooks/useDeposits.ts

```diff
@@ -47,19 +52,28 @@ export const useDeposits = (depositPageParams: PageParams) => {
   const l1Provider = useMemo(() => l1.provider, [l1.network.id])
   const l2Provider = useMemo(() => l2.provider, [l2.network.id])
 
+  const {
+    layout: { isTransactionHistoryShowingSentTx }
+  } = useAppContextState()
+
   const {
     app: {
       arbTokenBridge: { walletAddress }
     }
   } = useAppState()
 
+  const additionalQueryParams = isTransactionHistoryShowingSentTx
```

Comment:
NIT: additionalQueryParams gives the impression of being linked to app's query params somehow (useArbQueryParams). Same in withdrawals.

```suggestion
  const additionalSubgraphQueryParams = isTransactionHistoryShowingSentTx
```